itou/users/tests/test_models.py

```diff
@@ -105,6 +105,19 @@ def test_prescriber_of_authorized_organization(self):
         prescribermembership = PrescriberMembershipFactory(user=prescriber, organization__is_authorized=True)
         assert prescriber.is_prescriber_of_authorized_organization(prescribermembership.organization_id)
 
+    def test_is_orienter(self):
+        job_seeker = JobSeekerFactory()
+
+        assert not job_seeker.is_orienter
```

Comment:
_nit_: `assert job_seeker.is_orienter is False` comme ça le type est testé au passage.

Et si tu test les candidats faudrait aussi tester les autres `kind` ;).

Reply:
Pas faux. Merci !